diff --git a/FRIDA.stmx b/FRIDA.stmx
index 50900c0..a2cef7d 100644
--- a/FRIDA.stmx
+++ b/FRIDA.stmx
@@ -79,7 +79,15 @@
 	<isee:number_format_extenders/>
 	<default_format scale_by="1"/>
 	<data>
+		<export frequency="on_demand" resource="r../Data/uncertainty_analysis_exported_variables.csv" interval="DT" precomputed="false" isee:format="numbers">
+			<list>r../Data/varsForExport.txt</list>
+		</export>
 		<import isee:overwrite="false" isee:timevarying="true" resource="r../Data/frida_input_data.csv"/>
+		<import isee:overwrite="false" orientation="horizontal" resource="r../Data/uncertainty_analysis_paramter_values.csv"/>
+		<import isee:overwrite="false" orientation="horizontal" resource="r../Data/ClimateSTAOverride.csv"/>
+		<import isee:overwrite="false" orientation="horizontal" resource="r../Data/policyParameters.csv"/>
+		<import isee:overwrite="false" orientation="horizontal" resource="r../Data/climateFeedbackSwitches.csv"/>
+		<import isee:overwrite="false" isee:timevarying="true" resource="r../Data/ClimateSTAOverrideTS.csv"/>
 	</data>
 	<isee:sensi_specs_list active_index="0">
 		<isee:sensi_specs name="Climate Sensitivity" method="grid">
